RELEASE.md

```diff
@@ -0,0 +1,97 @@
+# Release
+
+Follow these steps to release a new version of spyder-vim.
+
+In the commands below, replace `X.Y.Z` with the release version when needed.
+
+**Note**: We use `pip` instead of `conda` here even on Conda installs, to ensure we always get the latest upstream versions of the build dependencies.
+
+## PyPI and GitHub
+
+You will need to have a local clone of the repo. The following steps supose a repo setup from a fork with and `upstream` remote pointing to the main `spyder-vim` repo
```

Comment:
You will need to have a local clone of the repo. The following steps suppose a repo setup from a fork with and `upstream` remote pointing to the main `spyder-vim` repo